packages/test/test-end-to-end-tests/src/test/pendingOps.spec.ts

```diff
@@ -86,7 +86,7 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
     // Observing non-deterministic timeouts in CI only.
     // Temporarily disabling pending investigation:
     //
-    // https://github.com/microsoft/FluidFramework/issues/6006
+    // https://github.com/microsoft/FluidFramework/issues/6157
     before(function() {
```

Comment:
Just updating with a more specific issue #:

#6006 is tracking the fact that RealSvc on Tinylicious is disabled.
#6157 is tracking is tracking this specific suite.